deoldify/filters.py

```diff
@@ -14,12 +14,9 @@
 
 class IFilter(ABC):
     @abstractmethod
-    def filter(
-        self, orig_image: PilImage, filtered_image: PilImage, render_factor: int
-    ) -> PilImage:
-        pass
-
-
+    def filter(self, orig_image:PilImage, filtered_image:PilImage, render_factor:int)->PilImage:
```

Comment:
Why this was changed?

Reply:
It has not sense to split the arguments if you are not using some style guide as PEP8.

Reply:
I have changed it anyway, but there is some standards about the style guide of the code, as PEP8. Actually, this line as others, don't follow thee PEP8 standard